tools/distpartitioning/parmetis_wrapper.py

```diff
@@ -24,6 +24,11 @@ def check_dependencies():
         mpi_install
     ), "Could not locate the following dependency: MPI. Please install it and try again."
 
+    dgl_path = os.environ.get("DGL_HOME")
```

Comment:
`os.environ.get("DGL_HOME", "")`. `os.path.isdir()` does not work with `None`

Reply:
Done.